contracts/neofs/contract.go

```diff
@@ -369,7 +369,7 @@ func Cheque(id []byte, user interop.Hash160, amount int, lockAcc []byte) {
 //
 // This method produces Bind notification. This method panics if keys are not
 // 33 byte long. User argument must be a valid 20 byte script hash.
```

Comment:
now this requirement follows from the type, mb drop?

Reply:
Well, keys are type-specific for a long time as well and yet there is something about their length above. This doesn't make much sense to me, but doesn't really hurt either.